company-profiles/cyberwhale.md

```diff
@@ -0,0 +1,41 @@
+# Cyber Whale
+
+## Company blurb
+
+SaaS Solutions for Fintech and eCommerce.
+We value quality, responsibility and holistic approach to creating products.
+
+## Company size
+
+15+
+
+## Remote status
+Based in Republic of Moldova, working with customers from US, UK, EU, Israel.
```

Comment:
This part is for the remote status of the company - do you hire people outside of Moldova? Do you know hire in certain regions?

Reply:
We hire remotely in EU and Eastern Europe 👍 will update this section